starwars/src/Components/CharCard/CharCard.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import './CharCard.css';
+import { Card, CardImg, CardText, CardBody, CardTitle, CardSubtitle, Button, UncontrolledCarousel } from "reactstrap";
+
+
+export const CharCard = (props) => {
```

Comment:
Would be a good spot to use object destructuring, since `props` is just an object afterall.

```
export const CharCard = ({ post }) => {
```

Then on line 9, reference `post.map()`